src/Congo.WebApi/Congo.WebApi.csproj

```diff
@@ -6,15 +6,17 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="MediatR" Version="9.0.0" />
-    <PackageReference Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="9.0.0" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.10" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.10" />
+    <PackageReference Include="Mapster" Version="7.2.0"/>
```

Comment:
You might have noticed that it removed the space at the end of the tag, which caused the diff to be much larger than needed.

What approach/tool did you use to add these references? Because when I added a new package, the entire file was reformatted to include the space, both using VS NuGet package manager and the CLI `dotnet package add`. 

I suggest we stick to the standard, which is including the space.